src/Tracy/Panel.php

```diff
@@ -18,26 +19,28 @@ public function __construct(SnapshotStack $snapshotStack)
 
 	public function getTab(): string
 	{
-		if ($this->snapshotStack->getSnapshots() === []) {
-			return '';
-		}
-
-		// phpcs:disable
-		$snapshotStack = $this->snapshotStack;
-
-		ob_start();
-		require __DIR__ . '/templates/tab.phtml';
-		return (string) ob_get_clean();
+		return Helpers::capture(function (): void {
+			// phpcs:disable
+			$totalTime = $this->snapshotStack->getTotalTime();
+			$count = $this->snapshotStack->getNumberOfSnapshots();
+			require __DIR__ . '/templates/tab.phtml';
+		});
 	}
 
-	public function getPanel(): string
+	/**
+	 * @return string|null
```

Comment:
Phpdoc is not needed, is it?

Reply:
Sure the PHPdoc is not needed. The return type was changed to string or null

if is null nothing will be displayed